### ISSUE TITLE
The private folder

### ISSUE THREAD

**Issue by shainova** [Ecological-Complexity-Lab/emln_package#95]
@Noa164 Do we need this? If not, remove it

**Comment by Noa164** [comment]
@shainova yes its where the markdown files for each network is stored. and also the file where we create the figures.

**Comment by shainova** [comment]
@Noa164 So change its name to soemthing more meaningful and dont forget to include it in the ignore. Then you can close the issue

**Comment by Noa164** [comment]
Changed name to "development"
